spec.txt

```diff
@@ -4080,6 +4080,18 @@ Here are some list items that start with a blank line but are not empty:
 </ul>
 ````````````````````````````````
 
+The number of spaces following the list marker doesn't affect the
```

Comment:
Could this be reworded?  Obviously, the number of spaces following the list marker does, in general, affect the indentation required.  It just doesn't when the line contains only spaces.